Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM maven:3.6.3-jdk-11-slim
+COPY pom.xml /usr/pom.xml
+COPY client/pom.xml /usr/local/pom.xml
+COPY client/src /usr/local/src
+WORKDIR /usr/local
+RUN mvn clean install
```

Comment:
Ici, c'est la commande `mvn clean package` qu'il faut utiliser.

Voici des articles parlant de la différence entre la commande `mvn package` et `mvn install` :
- https://stackoverflow.com/questions/16602017/how-are-mvn-clean-package-and-mvn-clean-install-different
- http://maven.apache.org/guides/introduction/introduction-to-the-lifecycle.html

Reply:
Merci pour la doc Loic. Je vais voir ça.